src/main/java/me/desht/pneumaticcraft/common/block/entity/ManualCompressorBlockEntity.java

```diff
@@ -0,0 +1,94 @@
+package me.desht.pneumaticcraft.common.block.entity;
+
+import me.desht.pneumaticcraft.api.pressure.PressureTier;
+import me.desht.pneumaticcraft.common.core.ModBlockEntities;
+import me.desht.pneumaticcraft.common.core.ModSounds;
+import me.desht.pneumaticcraft.common.network.DescSynced;
+import me.desht.pneumaticcraft.lib.PneumaticValues;
+import net.minecraft.core.BlockPos;
+import net.minecraft.core.Direction;
+import net.minecraft.sounds.SoundSource;
+import net.minecraft.world.level.block.entity.BlockEntityType;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraftforge.items.IItemHandler;
+
+public class ManualCompressorBlockEntity extends AbstractAirHandlingBlockEntity {
+    private final int airPerPumpCycle = PneumaticValues.PRODUCTION_MANUAL_COMPRESSOR;
+    private final int effectiveVolume = airHandler.getVolume() * 5;
+    public int pumpCycleProgress = 0;
+
+    // Used to determine how difficult each pump cycle step is
+    // Needs to be synced to client for pumpbar rendering
+    @DescSynced
+    public int storedAir = 0;
+
+    ManualCompressorBlockEntity(BlockEntityType<?> type, BlockPos pos, BlockState state, PressureTier tier, int volume) {
+        super(type, pos, state, tier, volume, 0);
+    }
+
+    public ManualCompressorBlockEntity(BlockPos pos, BlockState state) {
+        this(ModBlockEntities.MANUAL_COMPRESSOR.get(), pos, state, PressureTier.TIER_ONE, PneumaticValues.VOLUME_MANUAL_COMPRESSOR);
+    }
+
+    @Override
+    public void tickServer() {
+        super.tickServer();
+        storedAir = airHandler.getAir();
```

Comment:
What precision does the client actually need here?  Sync'ing this every time it changes could lead to a lot of chatter.  Clamping the sync'd value to the nearest X will reduce traffic (and thus lag).